### ISSUE TITLE
Stochastic Gradient Descent and Gradient Descent Used in Applied Field

### ISSUE THREAD

**Issue by OPUS-Lightphenexx** [OPUS-Lightphenexx/Python-Machine-Learning-Methods#5]
Examples like Optimization of linear regression loss function Using Gradient Descent.
Finding the Best value when dealing with multiple datasets.
